dev-vcs/tig/tig-2.2.1.recipe

```diff
@@ -4,18 +4,24 @@ Tig functions mainly as a git repository browser. It can also help in the \
 staging changes for commit at chunk level and act as a pager for output \
 different git commands."
 HOMEPAGE="http://jonas.nitro.dk/tig/"
```

Comment:
Cool, I guess this means we have green lights for having more then one website in HOMEPAGE when it is used appropriately.
Thanks @mmuman!